core/src/main/java/org/frankframework/jdbc/datasource/FrankResources.java

```diff
@@ -100,6 +125,40 @@ private DataSource loadDataSourceThroughDriver(Class<?> clazz, String url, Prope
 		return dmds;
 	}
 
+	@SneakyThrows
+	private MqttClient loadMqttClient(String url, String authAlias, String username, String password, Properties properties) {
```

Comment:
Ik zou dit allemaal in de MqttClientFactory verwachten. En die past ook beter thuis in messaging.